### PR TITLE
Add support for mouse forward/back button in Help

### PR DESCRIPTION
**Pull request by neoedmund**
Add support for mouse <kbd>forward</kbd> <kbd>back</kbd> buttons in Help

### CODE REVIEW

**Comment by GregoryLundberg** [comment]
Thank you. Much more readable.

For future reference, you could have used an interactive rebase to merge the two commits into one; then force-pushed your branch to update the PR, avoiding the need to create a new one and close the old one.

**Comment by CelticMinstrel** [comment]
I'd like to see this become a proper hotkey, but basic support at least is a good start.

File: src/help/help_browser.cpp [hunk 1]
```diff
@@ -168,6 +168,27 @@ void help_browser::handle_event(const SDL_Event &event)
 				}
 			}
 		}
+		else {
+			// ERR_NG <<"MB:" << (int) mouse_event.button << "\n"; 			
+			bool mouse_back = false;
```
Comment (Vultraz) [inline src/help/help_browser.cpp]: You can simplify the assignment here:
```
const bool mouse_back = !back_button_.hidden() && mouse_event.button == SDL_BUTTON_X1;
```
Same for `mouse_forward`.

Comment (neoedmund) [reply src/help/help_browser.cpp]: Yeah. I made a new push 2b2f434 . Is there anything else need to do with git operation?

**Comment by Vultraz** [comment]
Looks good now. Could you squash the two commits, and I’ll merge? Thanks for working on this :)

**Comment by neoedmund** [comment]
Vultraz, could you tell me what exactly should I do to "squash the two commits", because I'm not familiar with the process.

**Comment by Vultraz** [comment]
Ah, it's ok, I'll use the Squash button here.

**Comment by GregoryLundberg** [comment]
He was referring to doing a local interactive rebase and force-pushing to your feature branch. Since you had two commits, the following would do it:

```
git rebase --interactive HEAD~2
git push -f
```
The interactive rebase presents a test file in an editor. The comments at the bottom remind the options. For this one, you'd change the second command to 'fixup', save and exit. The same process would have merged your whitespace revert in the old PR.

**Pull request merged**